content/p/vulnerability-disclosure-policy.md

```diff
@@ -0,0 +1,72 @@
+
+## Example Report
+
+```text
+- **Title**: Failed to start OSBuild Composer
+
+- **Environment**: AlmaLinux 8, Platform: X86_64, OS Version: 8.5
+
+- **Description**:
+    I am unable to start the OS build composer. When I try to start the `osbuild-composer.service`, it stops immediately:
+
+    [root@localhost ~]# systemctl status osbuild-composer.service
+    ● osbuild-composer.service - OSBuild Composer
+        Loaded: loaded (/usr/lib/systemd/system/osbuild-composer.service; enabled; vendor preset: disabled)
+        Active: failed (Result: exit-code) since Tue 2022-02-08 00:18:43 EST; 32min ago
+        Process: 1657 ExecStart=/usr/libexec/osbuild-composer/osbuild-composer (code=exited, status=1/FAILURE)
+        Main PID: 1657 (code=exited, status=1/FAILURE)
+
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: osbuild-composer.service: Failed with result 'exit-code'.
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: osbuild-composer.service: Service RestartSec=100ms expired, scheduling restart.
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: osbuild-composer.service: Start request repeated too quickly.
+    Feb 08 00:18:43 localhost.localdomain systemd[1]: Failed to start OSBuild Composer.
+
+- **Steps to Reproduce**:
+    1. Install AlmaLinux 8.4 and update to 8.5.
+    2. Run `sudo dnf install osbuild-composer composer-cli cockpit-composer bash-completion`.
+    3. Run `sudo systemctl enable --now osbuild-composer.socket`.
+    4. Try running `composer-cli blueprint list` or start the image builder service in Cockpit.
+
+- **Expected Result**: The OS build composer should start without issues.
+
+- **Actual Result**: `osbuild-composer.service` stops immediately with an exit code error.
+
+- **Severity**: Urgent
+```
+
+## Where to Report
+
+- For any issue that requires a coordinated release, send your report to [security@almalinux.org](mailto:security@almalinux.org) directly so we can coordinate a responsible patch and release.
+
+- For issues that are directly related to the AlmaLinux operating system itself and do not require coordinated disclosure, please send your report to [bugs.almalinux.org](https://bugs.almalinux.org), this ensures that your report is received by the right people.
+
+- For **non OS-related** reports (.e.g Elevate, almalinux.org website, etc), open an issue on the GitHub repo for that part of the project. 
+
+#### Report using GitHub issues
+
+To report a vulnerability via GitHub issues, click on the `Issues` tab at the top of any repository and then click on the `New issue` button, then click on the `Report a vulnerability` button and fill out the form.
```

Comment:
If not wrong, you need to click on `Security` and then on `Report a vulnerability`. Also, I believe that we need to enable `Private vulnerability reporting` in every repo. Wondering if it can be done at organization level

Reply:
This is referring to the new GitHub issue template. (In this PR and if accepted I would send a PR to all of our repos)

Reply:
oh, I see that you added a template for those that don't require to be reported in private.
This process won't work in repos that don't have the issue template provided in the PR, and from my humble opinion, I'd prefer that every security-related report goes private by default.

Any thoughts?

Reply:
@bennyvasquez I think @javihernandez has made a good point, we should be using this method instead of creating a GitHub issue template IMO.

https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository

What do you think? I can update the page as soon as I get home to match.

Reply:
I like @javihernandez's idea as well.

Reply:
I don't like limiting reporting to github, because not everyone has a github account, but everyone is willing to send an email. IIRC the need for a GH account was actually cited as a reason not to do that earlier this year when we were discussing it... somewhere.

Reply:
Because this is not directly connected to this PR, let's put the "report via GitHub" section aside and add it once this is settled.

Once you approve this I would remove this section for now, and I think we need to create an issue dedicated to this matter and add to it to restore the section.

Reply:
Love that! Good job catching the scope creep. <3

Reply:
Alright then, I opened an Issue on AlmaLinux/,github, https://github.com/AlmaLinux/.github/issues/12 I hope this was the right place for that. :upside_down_face: